packages/contracts/src/hash-consensus/hash-consensus.constants.ts

```diff
@@ -0,0 +1,8 @@
+import { CHAINS } from '@lido-nestjs/constants';
+
+export const HASH_CONSENSUS_TOKEN = Symbol('hashConsensus');
+
+export const HASH_CONSENSUS_ADDRESSES = {
+  [CHAINS.Mainnet]: '0xD624B08C83bAECF0807Dd2c6880C3154a5F0B288',
+  [CHAINS.Goerli]: '0x8d87A8BCF8d4e542fd396D1c50223301c164417b',
```

Comment:
We have 2 hash consensus contracts: for accounting oracle and validator exit bus oracle. Could you add both of them?

https://docs.lido.fi/deployed-contracts/#lido-v2-oracle-contracts